pkg/ctl/scale/nodegroup_test.go

```diff
@@ -75,6 +86,26 @@ var _ = Describe("scale", func() {
 				args:  []string{"nodegroup", "ng", "--cluster", "dummy", "--nodes", "2", "--nodes-min", "1", "--nodes-max", "1"},
 				error: fmt.Errorf("number of nodes must be within range of min nodes and max nodes"),
 			}),
+			Entry("with config file and no name flags", invalidParamsCase{
+				args:  []string{"nodegroup", "-f", "../cmdutils/test_data/scale-ng-test.yaml"},
+				error: fmt.Errorf("--name must be set"),
+			}),
+			Entry("with config file and nodes flags", invalidParamsCase{
+				args:  []string{"nodegroup", "-f", "../cmdutils/test_data/scale-ng-test.yaml", "--nodes", "2"},
```

Comment:
The convention is that if one uses the config file the other CLI flags are incompatible so I would make all these flags incompatible with the config file and expect the user to set them in the appropriate field in the yaml.

Reply:
@martina-if Is this one https://github.com/weaveworks/eksctl/pull/2257/files#diff-4976b4e216739d1a8ce8733f6c089b52R14 good enough, or there is something else that I could have missed ?